index.html

```diff
@@ -218,11 +218,7 @@ <h3 id="streaming-document-examples">Examples</h3>
               <h3 id="streaming-document-examples-valid">Valid Examples</h3>
               <pre class="example" title="Valid streaming document with @context, @id, and other properties">
                 {
-                  "@context": {
-                    "name": "http://schema.org/name",
-                    "url": "http://schema.org/url",
-                    "image": "http://schema.org/image"
-                  },
+                  "@context": "http://schema.org/",
```

Comment:
I made the context more verbose to keep the examples fully standalone.
But I don't have a strong opinion on this, so I'm fine with changing it.

Reply:
I understand that, but so doing, you also changed their semantics, to something that may be surprising/disturbing to some readers. Hence my proposal